src/Shared/UnitTests/MockLogger.cs

```diff
@@ -416,7 +416,7 @@ static private ResourceManager EngineResourceManager
             {
                 if (s_engineResourceManager == null)
                 {
-                    s_engineResourceManager = new ResourceManager("Microsoft.Build.Resources.Strings", typeof(ProjectCollection).GetTypeInfo().Assembly);
+                     s_engineResourceManager = new ResourceManager("Microsoft.Build.Strings", typeof(ProjectCollection).GetTypeInfo().Assembly);
```

Comment:
Scratch that, you're right. I had gotten temporarily confused. Even though the .resource file name is `<AssemblyName>.<NeutralResxName>.resources`, the CLR strips away  the `.resources` part of the string.